frontend/src/utilities/valueUnits.ts

```diff
@@ -31,7 +31,7 @@ export const splitValueUnit = (
   options: UnitOption[],
 ): [value: number, unit: UnitOption] => {
   const match = value.match(/^(\d*\.?\d*)(.*)$/);
-  if (!match) {
+  if (!(match && match[1])) {
```

Comment:
With this change, the line after the `if` statement can never fallback to `1` because `match[1]` must can only be digits or decimal separated digits according to the regex. 
`const newValue = Number(match[1]) || 1; // avoid NaN`
Update to:
`const newValue = Number(match[1]);`